integrationTests/cypress/support/commands.js

```diff
@@ -160,6 +160,23 @@ Cypress.Commands.add('waitForSave', () => {
     });
 });
 
```

Comment:
For these new commands could we have an array/object which contains the nav items then map over that to create the commands? Might make it easier to add/remove nav items if the app changes.